lib/tasks/convert.rake

```diff
@@ -159,6 +159,6 @@ namespace :db do
     end
 
     desc 'Migrate MySQL (prod) to PostgreSQL (dev). Deletes ALL DATA in the development database.'
-    task :mysql2pgsql => :prod2dev
+    task :mysql2pgsql => ['prod2dev', 'db:sequence:reset']
```

Comment:
can we ensure the second task runs in the develop environment?

Reply:
Well I don't think so this will be an easy or clean task. And that's is indeed my initial concern I had: doing crazy Rake-Rails stuff just because to prevent user from running three commands instead of one. I don't believe this is worth it.

Reply:
what if we run the sequence reset inside the prod2dev task? we already iterate over the tables in both databases, somewhere around https://github.com/theforeman/foreman/blob/develop/lib/tasks/convert.rake#L152?

Reply:
Added the `ActiveRecord::Base.connection.reset_pk_sequence!(t)` statement to the `prod2dev` instead.